recipe/build.sh

```diff
@@ -1,5 +1,30 @@
 set -x
 
+source $RECIPE_DIR/get_cpu_arch.sh
+
+mkdir gcc-jit
+pushd gcc-jit
+../gcc/configure \
+    --host=$HOST \
+    --target=$HOST \
+    --enable-host-shared \
+    --enable-languages=jit \
+    --disable-bootstrap \
+    --disable-multilib \
+    --enable-libquadmath \
+    --enable-libquadmath-support \
+    --enable-long-long \
+    --disable-libssp \
+    --disable-libmudflap \
+    --enable-gold \
+    --disable-nls \
+    --prefix=$PREFIX/lib/emacs/jit
+make -j"${CPU_COUNT}"
+make install-strip
+cp -s "$PREFIX"/lib/emacs/jit/bin/x86_64-conda-linux-gnu-gcc-"$GCC_VERSION" "$PREFIX"/bin/
```

Comment:
Does this work on Mac too? This seems to be linux specific, but this code isn't in an `if uname == linux` block.

Reply:
I knew clang is preferred on mac, but didn't realize gcc will not build on osx. I move the JIT stuff into the non-Darwin branch.

There are still some hurdles on linux64 azure as well, I will look into this.

Reply:
I'm curious if the libgccjit stuff is even known to work at all on Mac? I would be interested in having it if it's possible, but I've only ever heard of people using it on Linux.

Reply:
Supposedly it can be done and Homebrew does it https://old.reddit.com/r/emacs/comments/qehd3c/build_emacs_with_native_compilation_in_macos/?rdt=61773. I haven't tested it. 

Of course, we don't need to try to get it working here. There's also an additional complication with mac that the ARM package is cross-compiled (is that still necessary?).

Reply:
Linux version builds and runs with clang just fine, so this should be doable indeed, but I don't have a Mac and the azure build is too slow.